code/src/java/pcgen/cdom/content/ProcessCalculation.java

```diff
@@ -62,38 +63,26 @@ public ProcessCalculation(T object, BasicCalculation<T> calc,
 		this.formatManager = fmtManager;
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
+	@SuppressWarnings("unchecked")
```

Comment:
No, this is a bad change.
This logically changes the warning that is suppressed, and cna can introduce errors.  It should stay down on the assignment of "input"